src/services/upower/org.freedesktop.UPower.Device.xml

```diff
@@ -0,0 +1,17 @@
+<node>
+    <interface name="org.freedesktop.UPower.Device">
+		<property name="Type" type="u" access="read"/>
+		<property name="PowerSupply" type="b" access="read"/>
+		<property name="Energy" type="d" access="read"/>
+		<property name="EnergyFull" type="d" access="read"/>
+		<property name="EnergyRate" type="d" access="read"/>
+		<property name="TimeToEmpty" type="x" access="read"/>
+		<property name="TimeToFull" type="x" access="read"/>
+		<property name="Percentage" type="d" access="read"/>
+		<property name="Temperature" type = "d" access="read"/>
```

Comment:
Add a prop for this

Reply:
I removed this property since my device does not support it meaning I cannot test it to see if it works on DisplayDevice as well as normal devices. I am willing to add it back, but I don't see it as being necessary for a simple battery service.

Reply:
yeah don't worry about it then